org.baedert.liblistbox.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id" : "org.baedert.liblistbox",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "demo",
+    "finish-args" : [
+        "--socket=wayland",
+        "--share=network"
```

Comment:
network is unneeded, right? Or is that because something does the meson subproject stuff?